tests/utilities_02.mpirun=2.output

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
The name of the file should be

`tests/utilities_02.with_mpi=on.mpirun=2.output`

because apparently Travis does not understand it does not have mpi...